doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs.rst

```diff
@@ -0,0 +1,253 @@
+.. _ncs_release_notes_2699_cs:
+
+|NCS| v2.6.99-cs Release Notes
+##############################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| v2.6.99-cs is a **customer sampling** release, tailored exclusively for participants in the nRF54L15 customer sampling program.
+Do not use this release of |NCS| if you are not participating in the program.
+
+The release is not part of the regular release cycle and is specifically designed to support early adopters working with the nRF54L15 device.
```

Comment:
Should we state its is SOC1 @barsok

Reply:
@barsok  @spresley

Reply:
I suggest we should avoid giving the name. The involved customers should know it is EngA/SOC1. @spresley ?